diff --git a/src/linux/gpio.c b/src/linux/gpio.c
index bb07f5a07a9f..b87b72083719 100644
--- a/src/linux/gpio.c
+++ b/src/linux/gpio.c
@@ -10,7 +10,7 @@
 #include <string.h> // memset
 #include <sys/ioctl.h> // ioctl
 #include <unistd.h> // close
-#include </usr/include/linux/gpio.h> // GPIOHANDLE_REQUEST_OUTPUT
+#include NIXOS_GPIO_H_INCLUDE_PATH
 #include "command.h" // shutdown
 #include "gpio.h" // gpio_out_write
 #include "internal.h" // report_errno
diff --git a/src/linux/main.c b/src/linux/main.c
index f9ea3f6daaa8..5f0b4dc6f089 100644
--- a/src/linux/main.c
+++ b/src/linux/main.c
@@ -4,7 +4,8 @@
 //
 // This file may be distributed under the terms of the GNU GPLv3 license.

-#include </usr/include/sched.h> // sched_setscheduler sched_get_priority_max
+#include NIXOS_SCHED_H_INCLUDE_PATH
+#include NIXOS_SCHED_TYPES_H_INCLUDE_PATH
 #include <stdio.h> // fprintf
 #include <string.h> // memset
 #include <unistd.h> // getopt
